### PR TITLE
[7003] Content diff current version fallback error

### PR DESCRIPTION
**Pull request by jvega190**
https://github.com/craftercms/craftercms/issues/7003
https://github.com/craftercms/craftercms/issues/5772

### CODE REVIEW

File: templates/web/diff.ftl [hunk 1]
```diff
@@ -108,27 +108,30 @@
...
+<#include "/static-assets/app/pages/legacy.html">
 <script>
-  $(document).ready(function(){
-    CStudioAuthoring.Service.getCurrentVersion(CStudioAuthoringContext.site, "${dir}", {
-      success: function(version) {
-        $('#current-version').append(version)
-      }
-    });
+  document.addEventListener("CrafterCMS.CodebaseBridgeReady", () => {
+    jQuery(function() {
```
Comment (rart) [inline templates/web/diff.ftl]: Probably not necessary to add the jquery ready function

**Review by rart — CHANGES_REQUESTED** [review]

File: templates/web/diff.ftl (hunk 1 above)
Comment (jvega190) [reply templates/web/diff.ftl]: Yeah it wasn't necessary, though it seems like it was delaying the script and causing the store to be ready, so I removed the jquery ready and now I subscribed to the store for it to be ready when calling the request (for it to have the needed tokens, etc)

File: templates/web/diff.ftl [hunk 2]
```diff
@@ -108,12 +108,15 @@
...
 
+<#include "/static-assets/app/pages/legacy.html">
 <script>
-  $(document).ready(function(){
-    CStudioAuthoring.Service.getCurrentVersion(CStudioAuthoringContext.site, "${dir}", {
-      success: function(version) {
-        $('#current-version').append(version)
-      }
+  document.addEventListener("CrafterCMS.CodebaseBridgeReady", () => {
+    window.craftercms.store$().subscribe(() => {
```
Comment (rart) [inline templates/web/diff.ftl]: For better race condition avoidance...
```ts
function init() {
  window.craftercms.store$().subscribe(() => {
    ...
  });
}
if (window.craftercms?.store$)
  init()
else 
  document.addEventListener("CrafterCMS.CodebaseBridgeReady", init)
```

**Review by rart — CHANGES_REQUESTED** [review]

**Review by rart — APPROVED** [review]
@jvega190 why draft?

**Pull request merged**